src/components/ListItemWithIcon.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import {TouchableHighlight, Text, View, Image, StyleSheet} from 'react-native';
+import Helpers from '../utils/helpers';
```

Comment:
you can use object deconstruction here:
`import {shortDesc} from ...`
then you can just use `shortDesc` instead of `Helpers.shortDesc` on line 14